api/src/test/java/org/pmiops/workbench/billing/BillingProjectBufferServiceTest.java

```diff
@@ -479,13 +489,65 @@ public void assignBillingProject_locking() throws InterruptedException, Executio
   @Test
   public void cleanBillingBuffer_creating() {
     DbBillingProjectBufferEntry entry = new DbBillingProjectBufferEntry();
+    entry.setFireCloudProjectName("foo");
+    entry.setStatusEnum(CREATING, this::getCurrentTimestamp);
+    entry.setCreationTime(getCurrentTimestamp());
+    billingProjectBufferEntryDao.save(entry);
+
+    doReturn(new BillingProjectStatus().creationStatus(CreationStatusEnum.CREATING))
+        .when(fireCloudService)
+        .getBillingProjectStatus(entry.getFireCloudProjectName());
+
+    CLOCK.setInstant(AFTER_CREATING_TIMEOUT);
```

Comment:
nit: "after creating timeout" can be parsed two ways. I initially read it as after (creating timeout), meaning after it was started, rather than after it was expired. Maybe `AFTER_CREATING_TIMEOUT_ELAPSED` would be clearer.

Reply:
Done